guides/common/modules/con_using-ipxe-to-reduce-provisioning-times.adoc

```diff
@@ -5,6 +5,13 @@ iPXE is an open-source network-boot firmware.
 It provides a full PXE implementation enhanced with additional features, such as booting from an HTTP server.
 For more information about iPXE, see link:https://ipxe.org[iPXE website].
 
+ifdef::satellite[]
```

Comment:
Just curious: Satellite-only? If the feature is to be removed from code, then it won't be available in any flavor, will it?

Reply:
The iPXE code is external from the Foreman code base (well except for provisioning templates and PXE loaders). For Satellite, iPXE is provided by RHEL OS packages. These iPXE packages will no longer be included in RHEL 10. For upstream users, it doesn't stop them from installing iPXE from somewhere else and continuing using it. I'm not aware that we will be removing the functionality from Foreman.

Reply:
Note that it mentions that they're removing a lot of drivers, not iPXE entirely:
>  Instead, iPXE now depends on the platform firmware to provide a NIC driver for the network boot. The /usr/share/ipxe/ipxe-snponly-x86_64.efi and /usr/share/ipxe/undionly.kpxe iPXE binary files are the part of the ipxe-bootimgs package and use the NIC driver provided by the platform firmware.

I'm not familiar enough with the matter to say for sure what this means for end users because I don't know how well supported the platform firmware is.

Reply:
Should there be a community board discussion regarding this major change? I guess, it will affect a lot of scenarios of users?

Reply:
> Should there be a community board discussion regarding this major change? I guess, it will affect a lot of scenarios of users?

This specific PR is only affecting Satellite and the change @Lennonka mentioned is what happens in EL10 so Foreman isn't the right project. To be explicit: Foreman has never packaged iPXE and always consumed it from somewhere else.